modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java

```diff
@@ -137,11 +203,13 @@ public void testGetMediaObjectAssetTags() throws Exception {
 	public void testGetPageItems() throws Exception {
 		String content = RandomTestUtil.randomString(10);
 
+		byte[] bytes = content.getBytes(StandardCharsets.UTF_8);
+
 		_addFileEntry(
 			_group.getGroupId(),
 			new MediaObjectImpl(
 				new BinaryFile(
-					new ByteArrayInputStream(content.getBytes(UTF_8)), 0L,
+					new ByteArrayInputStream(bytes), 0L,
```

Comment:
if bytes is used only here. don't declare the variable and use it here: content.getBytes(StandardCharsets.UTF_8)